### PR TITLE
🚀  Release v1.7.3

### PR DESCRIPTION
**Pull request by vishalanarase**
/area release

### CODE REVIEW

**Comment by adilGhaffarDev** [comment]
/hold
LGTM from my side. Holding so others can also review.

**Review by adilGhaffarDev — APPROVED** [review]
/approve

File: CHANGELOG/v1.7.3.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## :seedling: Others
+- clusterctl: Bump cert-manager to v1.15.0 (#10732)
+- Dependency: Bump to Go 1.21.11 (#10740)
+- Devtools: Use setup-envtest from CR main (05/2024) to use envtest binaries from CT releases (#10663)
+- e2e: Kind: bump to v0.23.0 (#10641)
+- e2e: Test: check for metadata.yaml when resolving releases to not try to use unreleased versions (#10645)
+- e2e: Test: improve output for ValidateResourceVersionStable by using BeComparable instead of Equal (#10654)
+- Security: Cleanup owner files (#10679)
+- Testing: Support MachinePools without MachinePoolMachines in clusterctl upgrade test (#10614)
+- Testing: Use kind as a secondary management cluster for clusterctl E2E tests (#10639)
```
Comment (sbueringer) [inline CHANGELOG/v1.7.3.md:33]:
```suggestion
```
Same comment as on the other PR (https://github.com/kubernetes-sigs/cluster-api/pull/10743#discussion_r1634537788)

/hold

Comment (sbueringer) [reply CHANGELOG/v1.7.3.md:33]: /hold cancel
Double checked all other PRs, the other ones are from release-1.6. But to be honest I don't have the time to do this :)

Comment (vishalanarase) [reply CHANGELOG/v1.7.3.md]: Updated the PR

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

**Comment by ader1990** [comment]
@vishalanarase Can you please check https://github.com/kubernetes-sigs/cluster-api/issues/10749?
Maybe some github actions were not run yet?

Thanks.

**Comment by adilGhaffarDev** [comment]
> @vishalanarase Can you please check #10749? Maybe some github actions were not run yet?
> 
> Thanks.

release is not published yet, only tag is created.

**Comment by ader1990** [comment]
As the name of this merged PR suggests, I assumed that the release should have been created, is really confusing.

**Comment by chrischdi** [comment]
Let's please not mix up discussions and continue in https://github.com/kubernetes-sigs/cluster-api/issues/10749

**Pull request merged**